.travis.yml

```diff
@@ -24,6 +38,7 @@ deploy:
     - dist/latest-mac.yml
     - dist/partyshare.busterlabs.xyz-*-mac.dmg
     - dist/partyshare.busterlabs.xyz-*-mac.zip
+    - dist/Partyshare.deb
```

Comment:
I believe the linux target _should_ follow the [`artifactName` scheme](https://github.com/BusterLabs/Partyshare/blob/b7c059d983eb8bb756fac994d7b687b0244ea2cd/package.json#L53), producing `partyshare.busterlabs.xyz-1.2.7-linux.tar.gz` and `partyshare.busterlabs.xyz-1.2.7-linux.deb` for the latest release?

Reply:
@bhstahl true, just noticed, see my comment above